src/cljs/owlet_ui/components/activity/comments.cljs

```diff
@@ -0,0 +1,15 @@
+(ns owlet-ui.components.activity.comments
+  (:require [reagent.core :as reagent]
+            [re-frame.core :as re]))
+
+(defn load-comments [activity]
+  (js/disqusReset
+    (get-in activity [:sys :id])
+    (clojure.string/replace (-> js/document .-location .-href) "#" "#!")
```

Comment:
FYI, i changed this line locally to `(-> js/document .-location .-href)` and then left a comment on [Bug Life](http://localhost:4000/#/activity/2iLyFzilTGsOGEEGWS2gAU) to see if it would show up in "Latest Discussions" with the correct activity link. it hasn't shown up yet